bin/publish_workflow_ui.sh

```diff
@@ -17,18 +17,21 @@ do
   if [ -f package/workflow-ui.json ]; then
     echo "Copying workflow-ui.json for $PACKAGE"
 
-    mkdir -p site/$PACKAGE
-    cp package/workflow-ui.json site/$PACKAGE/workflow-ui.json
+    PACKAGE_ID=$(cat package/workflow-ui.json | jq -r ".packageId // .package")
```

Comment:
When the `workflow-ui.json` contains `packageId` use that as the destination for the files rather than the page name. So `/surfaces/site-settings/@netlify/feature-package-pilot` would become `/surfaces/site-settings/feature-pilot`